### PR TITLE
Trackers: biz.heraldcorp.com

### PR DESCRIPTION
**Pull request by ghost**
### URL: `https://biz.heraldcorp.com/view.php?ud=20201222000287`
Screenshot:
Screenshot

### Network Logs
network1
network2

Domain | Header | Meaning
---- | -------- | ---------
`110.93.143.144:8080` | `WT.sr` | The size of user's display
`110.93.143.144:8080` | `WT.bs` | The size of user's browser window
`110.93.143.144:8080` | `WT.js` | Can the website use Javascript?
`110.93.143.144:8080` | `WT.ul` | Language
`110.93.143.144:8080` | `WT.naviver` | HTTP User-Agent
`herald.loginside.co.kr` | `bz` | HTTP User-Agent
`herald.loginside.co.kr` | `bv` | HTTP User-Agent
`herald.loginside.co.kr` | `os` | HTTP User-Agent
`herald.loginside.co.kr` | `dim` | The size of user's display
`herald.loginside.co.kr` | `ul` | Language

### Code
```
https://res.heraldm.com/js/logcollectscript_201312.js
https://res.heraldm.com/js/acecounter_V70.js
```
debugger1
debugger2

### CODE REVIEW

File: SpywareFilter/sections/specific.txt [hunk 1]
```diff
@@ -22,6 +22,9 @@
 ||nd.demdex.net/event?d_mid=$redirect=nooptext,important,domain=9now.com.au
 ||adc.nine.com.au/?token=$redirect=nooptext,important,domain=9now.com.au
 !
+||res.heraldm.com/js/acecounter_*.js
+||res.heraldm.com/js/logcollectscript_*.js
+||herald.loginside.co.kr/?
```
Comment (zloyden) [inline SpywareFilter/sections/specific.txt:27]: I guess it should be `||herald.loginside.co.kr^` in first-party trackers.

File: SpywareFilter/sections/tracking_servers.txt [hunk 2]
```diff
@@ -38,6 +38,7 @@
 !||i.snssdk.com/log/*
 !||i.snssdk.com/slardar/sdk.js
 !
+||110.93.143.144:8080^
```
Comment (zloyden) [inline SpywareFilter/sections/tracking_servers.txt:41]: Unfortunately, I don't see such this request.

Comment (piquark6046) [reply SpywareFilter/sections/tracking_servers.txt:41]: It looks strange. I can see that request.
Please search `logcollect.hci` at Network section of Developer Tools.

File: SpywareFilter/sections/specific.txt (hunk 1 above)
Comment (piquark6046) [reply SpywareFilter/sections/specific.txt]: Okay.
Updated at a5212e9d42d2b2144450de1f085e96e42c9951cc.

File: SpywareFilter/sections/tracking_servers.txt (hunk 2 above)
Comment (zloyden) [reply SpywareFilter/sections/tracking_servers.txt:42]: I can see it now. `||110.93.143.144^$third-party` is ok here.

Comment (piquark6046) [reply SpywareFilter/sections/tracking_servers.txt]: 48e5f2f50956d21bf8f34bf704233433b18ff30e

**Review by zloyden — APPROVED** [review]

**Pull request merged**